guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc

```diff
@@ -1,18 +1,28 @@
 [id="Registering_Hosts"]
 = Registering Hosts
 
-There are two main methods for registering a host to {ProjectServer} or {SmartProxyServer}:
+There are three main methods for registering hosts to {ProjectServer} or {SmartProxyServer}:
 
-* Download and install the consumer RPM (_server.example.com_/pub/katello-ca-consumer-latest.noarch.rpm) and then run Subscription Manager.
+. Generate a `curl` command from {Project} and run this command from unlimited number of hosts to register them using the global registration template.
+This method is suited for both freshly installed hosts and hosts that have been previously registered, for example, to {Project} 5 or another {ProjectX}.
++
+By using this method you can also deploy {Project} SSH keys to hosts during registration to {Project} to enable hosts for remote execution jobs.
+For more information about the remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs].
++
+By using this method you can also configure hosts with Red{nbsp}Han Insights during registration to {Project}.
+For more information about using Insights with {Project} hosts, see {ManagingHostsDocURL}using-insights-with-satellite-hosts[Monitoring Hosts Using Red Hat Insights].
+
+. Download and install the consumer RPM `_server.example.com_/pub/katello-ca-consumer-latest.noarch.rpm` and then run Subscription Manager.
 This method is suited for freshly installed hosts.
 
-* Download and run the bootstrap script (_server.example.com_/pub/bootstrap.py).
+. Download and run the bootstrap script (_server.example.com_/pub/bootstrap.py).
 This method is suited for both freshly installed hosts and hosts that have been previously registered, for example, to {Project} 5 or another {ProjectX}.
 
 You can also register Atomic Hosts to {ProjectServer} or {SmartProxyServer}.
 
 Use one of the following procedures to register a host:
 
+* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]
```

Comment:
Late question: Where this is supposed to link to? I am unable to find such id.

Reply:
This PR added a title Registering a Host to {ProjectName} Using the Global Registration Template with this ID at guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc.

Reply:
Additionally, if `git grep` does not yield any results, try without `_managing-hosts` because this looks like it could be the `_{context}`.

Reply:
`registering-a-host-to-project-using-the-global-registration-template_managing-hosts` was changed to `registering-a-host_managing-hosts`, since 2.5 version. See my fix for ansible doc: https://github.com/theforeman/foreman-documentation/pull/716

Reply:
Oh that was it, context was ruining my greps thanks :-)

Reply:
Looks like `guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc` no longer exists in master, need to find out why.

Reply:
Ok found it, it was renamed here: https://github.com/theforeman/foreman-documentation/pull/514/files